src/core/interfaces.ts

```diff
@@ -379,6 +379,7 @@ export namespace CoreChartProps {
   export interface ChartAPI {
     chart: Highcharts.Chart;
     highcharts: typeof Highcharts;
+    highlightSeries(series: Highcharts.Series): void;
```

Comment:
Let's add a basic unit test to this api in the `src/core/__tests__/chart-core-api.test.tsx` tests. I suggest to have two cases:
1. A line chart with 3 series. We highlight the middle series using the API and validate only that series is highlighted.
2. A pie chart with 3 segments - we highlight a middle segment using the API... here, we have a problem: the pie chart legend operates points, not series. So, potentially, we might want to call the API differently, e.g. `highlightChartItem` (aka event name in the chart-extra-highlight).